plugin/trino-prometheus/src/test/java/io/trino/plugin/prometheus/PrometheusQueryRunner.java

```diff
@@ -74,6 +74,7 @@ public static PrometheusClient createPrometheusClient(PrometheusServer server)
         config.setQueryChunkSizeDuration(new Duration(1, DAYS));
         config.setMaxQueryRangeDuration(new Duration(21, DAYS));
         config.setCacheDuration(new Duration(30, SECONDS));
+        config.setPrometheusReadTimeout(new Duration(10, SECONDS));
```

Comment:
No changes requested. I couldn't find the reason we want to set default values explicitly here, but I understand it's consistent with existing code. Let's simplify later.